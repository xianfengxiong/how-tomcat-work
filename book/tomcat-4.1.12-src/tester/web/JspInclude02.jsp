<%@ page contentType="text/plain" %>This is before the include
<jsp:include page="<%= request.getParameter(\"path\") %>" flush="false"/>
This is after the include

