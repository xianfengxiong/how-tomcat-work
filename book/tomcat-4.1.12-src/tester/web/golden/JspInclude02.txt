This is before the include
This is the include

This is after the include

